test/PlaneWaveBasis.jl

```diff
@@ -112,20 +112,21 @@ end
     @test length(G_vectors(fft_size)) == prod(fft_size)
     @test length(r_vectors(basis))    == prod(fft_size)
 
-    @test all(G_vectors(basis)      .== G_vectors(fft_size))
-    @test all(G_vectors_cart(basis) .== map(G -> model.recip_lattice * G,
+    # Using `isapprox` due to https://github.com/JuliaMolSim/DFTK.jl/pull/727#issuecomment-1246749288
+    @test all(G_vectors(basis)      .≈ G_vectors(fft_size))
+    @test all(G_vectors_cart(basis) .≈ map(G -> model.recip_lattice * G,
```

Comment:
@epolack Be very careful! This is not a test, this is an assignment!

Also the comment is out of sync.

Reply:
> Doesn't .≈ still do the assignation, but at ε accuracy?

Not sure what you mean. `=` does assignment, `==` equality comparison, `===` identity comparision.

> Also not sure what is out of sync.

You are no longer using `isapprox`.

Reply:
Yes it is, sorry!

Would you mind adding a very strong abstol (e.g. 1e-12 or less) and also take a pass on the alignment.